src/util-decode-mime.c

```diff
@@ -3162,7 +3162,7 @@ static int MimeBase64DecodeTest01(void)
         return 0;
 
     ret = DecodeBase64(dst, strlen(msg) + 1, (const uint8_t *)base64msg, strlen(base64msg),
-            &consumed_bytes, &num_decoded, BASE64_MODE_RFC2045);
+            &consumed_bytes, &num_decoded, ModeRFC2045);
```

Comment:
@jasonish do you know how cbindgen can export this as `Base64ModeRFC2045` ?

Reply:
> @jasonish do you know how cbindgen can export this as `Base64ModeRFC2045` ?

Something like this? https://github.com/OISF/suricata/pull/8417/commits/857f23c17e472e3ef715e4d03f2a5d50f1b92c07

Reply:
I love that :-)

Reply:
But it has this drawback which is why I didn't pursue it further then: https://github.com/OISF/suricata/pull/8417#issuecomment-1399210803

But with the library being a focus, we really need to start prefixing everything in global name space, so it should be revisited. Either having `cbindgen` do it, or just giving things a prefix in Rust when its not very rust-like.